midp/gfx.js

```diff
@@ -24,14 +24,7 @@ var currentlyFocusedTextEditor;
     };
 
     function setFullScreen(isFullScreen) {
-        var sidebar = document.getElementById("sidebar");
-        var header = document.getElementById("drawer").querySelector("header");
-        var canvas = MIDP.Context2D.canvas;
-        sidebar.style.display = header.style.display =
-            isFullScreen ? "none" : "block";
-        var headerHeight = isFullScreen ? 0 : header.offsetHeight;
-        canvas.height = MIDP.ScreenHeight - headerHeight;
-        canvas.dispatchEvent(new Event('canvasresize'));
+        MIDP.setFullScreen(isFullScreen);
```

Comment:
The `setFullScreen()` in gfx.js can be remove and call `MIDP.setFullScreen()` directly instead.